src/components/HomepageBanner.js

```diff
@@ -0,0 +1,224 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import { Button, Link } from '@newrelic/gatsby-theme-newrelic';
+import bannerForeground from '../images/bannerForeground.svg';
+import relicsAtWork from '../images/photo-relics-at-work-01.png';
+
+const HomepageBanner = () => {
+  return (
+    <div
+      css={css`
+        max-height: 415px;
+        position: relative;
+        overflow: hidden;
+        border-radius: 0.5rem;
+        padding-top: calc(415 / 1252 * 100%);
```

Comment:
massive nit-pick, but could we set the calculation numbers as variables somewhere? maybe at the top of this file? That way it's easier to tell where the numbers came from. **Disclosure** I can def be talked out of this if this would be a huge pain and majorly divergent to the current pattern

```suggestion
        max-height: ${MAX_HEIGHT_DESKTOP}
        padding-top: calc(${MAX_HEIGHT_DESKTOP} / ${MAX_WIDTH_DESKTOP} * 100%);
```

Reply:
we definitely can, this is actually a weird hack to make the div the banner lives in respect a certain aspect ratio. so more like MAX_HEIGHT_BANNER/MAX_WIDTH_DESKTOP